include/dxc/DXIL/DxilShaderModel.h

```diff
@@ -26,7 +26,7 @@ class ShaderModel {
 public:
   using Kind = DXIL::ShaderKind;
 
-  // Major/Minor version of highest shader model
+  // Major/Minor version of highest released shader model
```

Comment:
This change isn't accurate now.  Perhaps add the comment with "released" added before the block generated by `get_highest_released_shader_model()` instead.